demo/vl-rich-data-table.html

```diff
@@ -41,16 +42,25 @@ <h2>Basis rich data table</h2>
                         &lt;/template&gt;
                     &lt;/vl-rich-data-field&gt;
                 &lt;/vl-rich-data-table&gt;
+                &lt;script&gt;
+                    customElements.whenDefined('vl-rich-data-field').then(() =&gt; {
+                        document.querySelector('[data-vl-label="Tweede medewerker"]').renderer = (element, item) =&gt; {
+                            const medewerker = item.medewerkers[1];
+                            element.textContent = `${medewerker.firstName} ${medewerker.lastName}`;
+                        };
+                    });
+                &lt;/script&gt;
             </code>
         </pre>
         <div class="demo">
             <vl-rich-data-table id="rich-data-table"
-                data-vl-data='{"data": [{"id": 0, "name": "Project #1", "manager": {"firstName": "Pascal", "lastName": "Riquier"}, "medewerkers": [{"firstName": "Sander", "lastName": "Kleykens"}]}, 
-                	{"id": 1, "name": "Project #2", "manager": {"firstName": "Tom", "lastName": "Coemans"}, "medewerkers": [{"firstName": "Guy", "lastName": "Wauters"}]}]}'>
+                data-vl-data='{"data": [{"id": 0, "name": "Project #1", "manager": {"firstName": "Pascal", "lastName": "Riquier"}, "medewerkers": [{"firstName": "Sander", "lastName": "Kleykens"}, {"firstName": "Tom", "lastName": "Coemans"}]}, 
+                	{"id": 1, "name": "Project #2", "manager": {"firstName": "Tom", "lastName": "Coemans"}, "medewerkers": [{"firstName": "Guy", "lastName": "Wauters"}, {"firstName": "Tom", "lastName": "Coemans"}]}]}'>
                 <vl-rich-data-field data-vl-label="ID" data-vl-selector="id"></vl-rich-data-field>
                 <vl-rich-data-field data-vl-label="Naam" data-vl-selector="name"></vl-rich-data-field>
                 <vl-rich-data-field data-vl-label="Naam manager" data-vl-selector="manager.lastName"></vl-rich-data-field>
                 <vl-rich-data-field data-vl-label="Eerste medewerker" data-vl-selector="medewerkers.0.lastName"></vl-rich-data-field>
+                <vl-rich-data-field id="rich-data-table-field-renderer" data-vl-label="Tweede medewerker"></vl-rich-data-field>
```

Comment:
de id lijkt mij nie per se nodig als ge met de code hieronder em toch selecteert by label?

Reply:
Klopt, ik had het alleen aangepast in het code voorbeeld, ik kijk al uit om de vl-ui-demo branches te merken zodat we hiervan verlost zijn. Ik heb het aangepast.